agent/wsclient/client_test.go

```diff
@@ -36,6 +40,11 @@ import (
 
 const dockerEndpoint = "/var/run/docker.sock"
 
+// Close closes the underlying connection
+func (cs *ClientServerImpl) Close() error {
```

Comment:
Why is this implemented in the test file?

Reply:
`ClientServerImpl` is a partial implementation of the interface, which lacks the `Close` method. If this is not there, there'll be NPEs. I can document that to make it more clear.